src/components/mobile/mobile-sub-header.vue

```diff
@@ -1,10 +1,12 @@
 <template>
   <CollapseHeightAnimation class="x-layout-item" :isCollapsed="hasScrolledPastThreshold">
-    <LocationProvider location="predictive_layer">
-      <RelatedTags class="x-pb-16" />
-    </LocationProvider>
-    <div v-if="$x.query.search && !$x.redirections.length">
-      <MobileToolbar class="x-mb-16" />
+    <div>
```

Comment:
Hi! We could move the `"x-layout-item"` class to the container `<div>` to keep the structure more similar to desktop, but up to you!

Reply:
I have already tried to move the `"x-layout-item"` class to the `<div>`, but it makes the collapse not working properly. It collapses like in 2 steps instead of in 1.